apps/passport-client/components/screens/ScannedTicketScreens/PodboxScannedTicketScreen/sections/PodboxInstantCheckin.tsx

```diff
@@ -0,0 +1,72 @@
+import { Spacer } from "@pcd/passport-ui";
+import { useEffect } from "react";
+import { RippleLoader } from "../../../../core/RippleLoader";
+import { ErrorContainer, StatusContainer } from "../PodboxScannedTicketScreen";
+import { PodboxTicketActionErrorSection } from "./actions/PodboxTicketErrors";
+import { useExecuteTicketAction } from "./actions/useExecuteTicketAction";
+
+export function InstantCheckin({
+  ticketId,
+  eventId
+}: {
+  ticketId: string;
+  eventId: string;
+}): JSX.Element {
+  const executor = useExecuteTicketAction({
+    eventId,
+    ticketId,
+    action: {
+      checkin: true
+    }
+  });
+
+  useEffect(() => {
+    executor.execute();
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
+
+  if (executor.loading) {
+    return (
+      <>
+        <RippleLoader />
+        <Spacer h={8} />
+      </>
+    );
+  }
+
+  if (executor.result?.success) {
+    const value = executor.result.value;
+    if (value.success) {
+      // For the richer "action executor" system used when camera-scanning, we
+      // have different success messages. These are fetched from the pre-check
+      // API, which we do not use here. However, since we can only do check-in,
+      // it's OK that the message is hard-coded.
+      const successMessage = "Checked In";
+      return (
+        <>
+          <StatusContainer size="small">{successMessage}</StatusContainer>
+        </>
+      );
+    } else {
+      return <PodboxTicketActionErrorSection error={value.error} />;
+    }
+  } else if (executor.result?.error) {
+    const errorMessage = "Error Checking Ticket In";
```

Comment:
also here - "Error Giving Swag"

Reply:
https://github.com/proofcarryingdata/zupass/blob/ivan/swag-stats/apps/passport-client/components/screens/ScannedTicketScreens/PodboxScannedTicketScreen/sections/actions/checkin/CheckInActionSection.tsx#L52

Reply:
The problem is that this comes from the pre-check, and skipping the pre-check is the main feature of this PR :D